modules/ROOT/pages/setup.adoc

```diff
@@ -106,18 +110,22 @@ Known Limitations:
 Visualizer requires an additional header injection for standalone Mule and Runtime Fabric deployments.
 However, due to a potential performance impact for Mule 4.1.x, header injection is disabled by default.
 
+Enable the header injection in the Runtime Manager.
+
 [NOTE]
 For Runtime Fabric, you must enable the header injection at the app level. +
-For a standalone Mule deployment, you must enable the header injection at the server, not app, level. This affects all applications running on the server.
-
-Set the following property in either the server (for Mule standalone deployment) or app (for a Runtime Fabric deployment):
+For a standalone Mule deployment, you must enable the header injection at the server (not app) level. This affects all applications running on the server.
 
+. In Runtime Manager, to enable the header injection in the management page,  click *Applications* or *Server* in the menu on the left. 
+.. From the list view, select the hybrid app or server for which to enable the header injection, and click *Manage Application* or *Manage Server*. 
```

Comment:
Is it confusing to have a single indented bullet? Not clear whether this sends me to the deployment page, or some other page and I have to return to main Runtime Manager page and choose a deployment page instead of the management page (I know this might be clearer with the UI running side by side--just saying indenting this one item doesn't really make it any clearer than if it wasn't indented...